src/modeler/AutoRest.Swagger/Validation/NonAppJsonTypeWarning.cs

```diff
@@ -30,6 +30,11 @@ public override bool IsValid(string entity)
             => (entity.ToLowerInvariant().StartsWith(AppJsonType));
         
         /// <summary>
+        /// What kind of change implementing this rule can cause.
+        /// </summary>
+        public override ValidationChangesImpact ValidationChangesImpact => ValidationChangesImpact.ServiceImpactingChanges;
```

Comment:
Do we generate SDKs in AutoRest that can support non app json types? If yes, we can probably come up with another category (None) indicating no impact on either service or sdk ends.